src/renderer/components/modal/swap/SwapDialog.tsx

```diff
@@ -33,18 +33,21 @@ export default function SwapDialog({
   const classes = useStyles();
   const swap = useAppSelector((state) => state.swap);
   const [debug, setDebug] = useState(false);
-  const [openCancelAlert, setOpenCancelAlert] = useState(false);
+  const [openSuspendAlert, setOpenSuspendAlert] = useState(false);
   const dispatch = useAppDispatch();
 
   function onCancel() {
     if (swap.processRunning) {
-      setOpenCancelAlert(true);
+      setOpenSuspendAlert(true);
     } else {
       onClose();
       setImmediate(() => dispatch(swapReset()));
```

Comment:
**Suggestion:** Consider using a more descriptive name for the `onCancel` function. The name should reflect the function's purpose and the action it performs.
```suggestion
  function handleCancel() {
    if (swap.processRunning) {
      setOpenSuspendAlert(true);
    } else {
      onClose();
      setImmediate(() => dispatch(swapReset()));
    }
  }
```